### PR TITLE
Add example deployment

### PR DESCRIPTION
**Pull request by ddorgan**

### CODE REVIEW

**Comment by ddorgan** [comment]
Pending release on watr repo for `v1.2.0`

File: docs/deployment.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Ansible
+====================
+
+Ansible is a popular tool to detect if a host matches the current expected state and also apply any changes or updates if required. Lots of create pre-packaged modules exist in [ansible galaxy](https://galaxy.ansible.com/).
```
Comment (TriplEight) [inline docs/deployment.md:4]:
```suggestion
Ansible is a popular tool to detect if a host matches the current expected state and also apply any changes or updates if required. Lots of pre-packaged modules exist in [ansible galaxy](https://galaxy.ansible.com/). 
```

File: docs/deployment.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+Ansible
+====================
+
+Ansible is a popular tool to detect if a host matches the current expected state and also apply any changes or updates if required. Lots of create pre-packaged modules exist in [ansible galaxy](https://galaxy.ansible.com/). 
+
+The paritytech.chain_operations module is maintained on [github](https://github.com/paritytech/ansible-galaxy) and available via [ansible-galaxy]()
```
Comment (TriplEight) [inline docs/deployment.md:6]:
```suggestion
The `paritytech.chain_operations` module is maintained on [github](https://github.com/paritytech/ansible-galaxy) and available via [ansible-galaxy]()
```
is `[ansible-galaxy]()` supposed to be a link? Can't find it in galaxy https://galaxy.ansible.com/search?deprecated=false&keywords=paritytech.chain_operations&order_by=-relevance&page=1

**Review by TriplEight — CHANGES_REQUESTED** [review]

**Comment by NachoPal** [comment]
Can you please add a link to the guide in the README.md under "Guides" section?

**Pull request merged**